### PR TITLE
Add final step to PR process instructions

### PR DESCRIPTION
**Pull request by scogra17**
Added a celebration step that I'd previously forgotten.

### CODE REVIEW

**Comment by katarinarosiak** [comment]
That's awesome!

File: README.md [hunk 1]
```diff
@@ -8,3 +8,4 @@ A static site for practicing GitHub collaboration
 * Push your branch to the remote: `$ git push origin [branch name]`
 * Create a pull request on the GitHub UI
 * Wait for insightful comments from your teammates :)
+* Celebrate with your teammates
```
Comment (mabbason) [inline README.md]: Let's add some detail for the type of celebration. Then again sometimes I've been accused of being a party pooper

Comment (katarinarosiak) [reply README.md]: Love that Miles!

Comment (scogra17) [reply README.md]: Great idea -- did you have something in mind?

Comment (tdronkers04) [inline README.md]: feels like an exclamation point is warranted here

**Review by mabbason — APPROVED** [review]
I like the thumbs up idea

**Pull request merged**